lib/pure/collections/sets.nim

```diff
@@ -347,6 +347,17 @@ proc excl*[A](s: var HashSet[A], other: HashSet[A]) =
   assert other.isValid, "The set `other` needs to be initialized."
   for item in other: discard exclImpl(s, item)
 
+proc pop*[A](s: var HashSet[A]): A =
+  ## Remove and return an arbitrary element from the set.
+  ## Raises KeyError if the set is empty.
```

Comment:
Small nitpick: Next time please add an empty line in between these. They shouldn't be in the same paragraph.